content/visualization/visualization_1.md

```diff
@@ -49,6 +50,20 @@ client_x509_cert_url = "xxx"
 [bigquery]
 project_id = "xxx"
 dataset = "dbt_xxx_derived"
+
+[databricks]
+server_hostname = "xxx"
+http_path = "xxx"
+access_token = "xxx"
+
+[snowflake]
+user = "xxx"
+password = "xxx"
+account = "xxx"
+database = "xxx"
+schema = "xxx"
+warehouse = "xxx"
+role = "xxx"
```

Comment:
We should add links to some information about where to identify this stuff for each warehouse - asking for a friend.